src/JsFunctionsScanner.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace WP_CLI\I18n;
+
+use Gettext\Translations;
+use Gettext\Utils\JsFunctionsScanner as GettextJsFunctionsScanner;
+use Gettext\Utils\ParsedComment;
+use Peast\Peast;
+use Peast\Syntax\Node\Node;
+use Peast\Syntax\Node\VariableDeclaration;
+use Peast\Traverser;
+use Peast\Syntax\Node\CallExpression;
```

Comment:
All of these `Node\Something` imports are a case where I would prefer the use of relative namespaces.
So, have only 1 import `Peast\Syntax\Node` here, and then use relative namespaces like `Node\Identifier` or `Node\Literal` in the code.
This makes the import list less daunting, and it gives more context to the consuming code, where it is clear that these are all different types of nodes.

Reply:
Hmm, I now see that the library has a `Node\Node`, which is unfortunate and might make my above suggestion weird.

I usually build my code so that the interface is outside of the namespace with the implementations. This way, you can use `Node` as well as `Node\Literal`, ...

I'll let you decide what you find cleaner, but having a `Node\Node` inside the code is certainly not really elegant.